### DISCUSSION TITLE
TP7 - Enmienda ejercicio1 y captura rota de 'directorios'

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#181] [Preguntas y respuestas] [answered]
Hola profe! en la parte de consideraciones del tp7 la imagen que pusiste de ejemplo esta caida. Podrias explicarme a que se refiere la consigna con "directorios diferentes"?

edit: tambien esta incompleta la consigna del punto 1 
![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/134400606/cdb99c93-7003-4185-a90f-d4843253c46b)

**Comment by martinvilu** [comment, marked as answer]
Este ejercicio debe hacer algunas operaciones desde la línea de comandos y liberar el valor pedido.

```
$> ./puntuales.exe 2 + 4
6
```

Suma, resta, multiplicación y división. Usando operaciones lentas.


La parte de los directorios es simplemente que cada ejercicio este en un directorio separado:

![271378451-7b0361a1-f7d0-4bd5-9a24-8f9c1377bcaf](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/9f8c94d7-05ff-46e3-b459-2cd202957e8c)
